### DISCUSSION TITLE
Dark Energy vs "Timescape" cosmology

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#295] [Cosmology Models]
https://phys.org/news/2024-12-dark-energy-doesnt-lumpy-universe.html

Interesting challenge to LCDM:

>Both the Hubble tension and the surprises revealed by DESI are difficult to resolve in models which use a simplified 100-year-old cosmic expansion law—Friedmann's equation.

>This assumes that, on average, the universe expands uniformly—as if all cosmic structures could be put through a blender to make a featureless soup, with no complicating structure. However, the present universe actually contains a complex cosmic web of galaxy clusters in sheets and filaments that surround and thread vast empty voids.

...

>When the proposed timescape model was last tested in 2017, the analysis suggested it was only a slightly better fit than the ΛCDM as an explanation for cosmic expansion, so the Christchurch team worked closely with the Pantheon+ collaboration team who had painstakingly produced a catalog of 1,535 distinct supernovae.

>They say the new data now provides "very strong evidence" for timescape. It may also point to a compelling resolution of the Hubble tension and other anomalies related to the expansion of the universe.

**Comment by RedshiftDrift** [comment]
I've never heard of the "timescape model"...
"The model suggests that a clock in the Milky Way would be about 35 percent slower than the same one at an average position in large cosmic voids, meaning billions more years would have passed in voids." https://phys.org/news/2024-12-dark-energy-doesnt-lumpy-universe.html

***35% time dilation?***  That seems exaggerated! Let's check that.
The Milky Way has $10^{12}$ solar masses (virial mass) or $2 \times 10^{42}$ kg,
Earth distance to centre of Milky Way is 8 kpc = $2.5 \times 10^{20}$ m,
gravitational [time dilation](https://en.wikipedia.org/wiki/Gravitational_time_dilation) at Earth's position is

$$ \frac{t_0}{t_f} = \sqrt{1 - \frac{2GM}{rc^2}} = \sqrt{1 - \frac{2 (6.7\times 10^{-11}) (2 \times 10^{42}) }{(2.5 \times 10^{20})c^2}} = \sqrt{1 - 1.2\times 10^{-5}} = 0.999994 $$

Our clocks are 0.0006% slower than clocks away from the Milky Way.  Other galaxies distributed over the local cluster have a much smaller mass density, than the MW, so even going to a large cosmic void could not give much more than 0.0006% time dilation.

The 35% quoted in the article (or [38% quoted here](https://en.wikipedia.org/wiki/Inhomogeneous_cosmology#Timescape_cosmology)) is grossly in error.  The "timescape model" is 99.9994% :shit::shit:

No wonder nobody pays attention to this and that I've never heard of it...

**Reply by mikehelland** [reply]
I hadn't heard of it either. 

It is published in MNRAS so, at the very least, it's an example of an alternative cosmology (although, not radically alternative along the lines that ACG members would prefer).

https://academic.oup.com/mnrasl/article/537/1/L55/7926647?login=false

As far as those calculations are involved, I think you need to compare a local clock in the Milky Way to a clock millions of light years away in some void between galaxies.

**Reply by RedshiftDrift** [reply]
> I think you need to compare a local clock in the Milky Way to a clock millions of light years away in some void between galaxies.

I already did, that's what I meant by "even going to a large cosmic void could not give much more than 0.0006% time dilation".  There's simply not enough density of matter.

MNRAS does not always publish good papers...  You can even find nonsense in A&A.

**Reply by mikehelland** [reply]
If I'm not mistaken, your calculation is placing a mass equal to the Milky Way's at a point, and then placing us 8 kpc away from that point.

The timescape cosmology (of which I am not a proponent, mind you) says the universe is inhomogenous, but not that inhomogenous.

I think you need to take more into consideration, like the mass of the halo, and probably the local supercluster. In general relativity, I think that means perturbations. 

But digging a little deeper, it seems the concept of a backreaction (of which I'm not familiar) plays a pivotal role, and is controversial in and of itself:

https://www.reddit.com/r/cosmology/comments/qcjqfy/paper_discussion_timescape_cosmology/

I think it's somewhat noteworthy papers like this are coming out of mainstream journals, because I think it tells us that people are more and more willing to conjecture alternatives as the "crisis in cosmology" builds. They aren't going to go as far as to question the expansion of the universe, yet, not while alternatives that are still "in-bounds" keep being proposed.

**Reply by RedshiftDrift** [reply]
> ...then placing us 8 kpc away from that point.  The timescape cosmology (of which I am not a proponent, mind you) says the universe is inhomogenous, but not that inhomogenous.

Having us at 8 kpc from the centre of the Milky Way is the most inhomogeneous situation possible.  Everything else in Timescape Cosmology is more homogeneous, therefore 0.0006% time dilation is an upper limit.

> I think you need to take more into consideration, like the mass of the halo, and probably the local supercluster. In general relativity, I think that means perturbations.

The mass of the Milky Way I used includes baryons, halo and even dark matter.  Again, my number is an upper limit which is still way too small compared to the claimed 35%.

> the concept of a backreaction plays a pivotal role,

Backreaction is an idea, a model...  We know how lumpy the universe is from observations, and it's not lumpier than this part of the Milky Way I used in my *worst case* calculation.

> I think it's somewhat noteworthy papers like this are coming out of mainstream journals, because I think it tells us that people are more and more willing to conjecture alternatives as the "crisis in cosmology"...

Sure, except that when a good alternative-paper is published, it is lost in the large number of bad papers like this one and doesn't get noticed.

**Reply by mikehelland** [reply]
>Having us at 8 kpc from the centre of the Milky Way is the most inhomogeneous situation possible.

in the sense that all curvature in the model is produced by a mass a single point. Which I don't think is how they're doing their calculations. 

So what's this backreaction stuff about?

**Reply by RedshiftDrift** [reply]
> in the sense that all curvature in the model is produced by a mass a single point.

No, I approximate the Laplace equation for the mass density and potential, which is what they should be using.  Since the largest divergence of the gradient happens to be near a galaxy, my approximation is a good estimation of the correct value, i.e. it's certainly not off by a factor $10^5$!

**Comment by Francois-Zinserling** [comment]
> As far as those calculations are involved, I think you need to compare a local clock in the Milky Way to a clock millions of light years away in some void between galaxies.

Problem is that void signal is already redshifted.

**Reply by RedshiftDrift** [reply]
Moreover, the "void" is only an under-density by a factor of about two, so voids don't really do anything special to gravity and redshift.

**Comment by Vlada3333** [comment]
Interesting, thank you!
